ext/mysqli/mysqli.stub.php

```diff
@@ -278,7 +278,7 @@ public function stmt_init() {}
      * @return mysqli_result|false
      * @alias mysqli_store_result
      */
-    public function store_result(int $flags = 0) {}
+    public function store_result(int $mode = 0) {}
```

Comment:
I'm a bit conflicted about this change. Yes, there currently is only a single flag. Logically this is the MYSQLI_USE_RESULT | MYSQLI_STORE_RESULT | MYSQLI_STORE_RESULT_COPY_DATA bitset, just with this case hardcoding MYSQLI_STORE_RESULT, thus leaving only a single flag one can toggle.